hardware/Makefile

```diff
@@ -234,12 +234,12 @@ patch_NVMe:
 	fi
 
 action_hw: prepare_logs
-	@echo -e "[CONFIG ACTION HW....] start `date +"%T %a %b %d %Y"`";
+	@echo -e "[COMPILE ACTION HW...] start `date +"%T %a %b %d %Y"`";
```

Comment:
This step is a compile step only for HLS actions. For HDL actions it is actually meant as configuration step.

Reply:
ok got it so let's keep it as previously